types/fitbit-asap/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for fitbit-asap 1.2
+// Project: https://github.com/dillpixel/fitbit-asap
+// Definitions by: Jérémy Jeanson <https://github.com/JeremyJeanson>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export * from "./app";
+export { Asap, Options } from "./companion";
```

Comment:
This one is OK, but original package has no single entry export file at all.
Here in DT that is requirement,
Let me ask on Discord if this is OK (it will just fulfill linter checks, but export not a thing):
```suggestion
// Type definitions for fitbit-asap 1.2
// Project: https://github.com/dillpixel/fitbit-asap
// Definitions by: Jérémy Jeanson <https://github.com/JeremyJeanson>
// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped

import * as app from './app';
import * as compation from './companion';
```
So please do not commit this suggestion yet,

Reply:
I have to change this file

Reply:
I like this suggestion, although it would help to add a comment explaining that the reason there's only imports and no exports is that the intended usage is deep imports of `app` and `companion`.

Reply:
It was an error, the last version has 2 export.

From the app who used this module, it is really special:
- The module is use by 2 applications
- the client "app" use the app.d.ts.
- the mobile "companion" use the companion.d.ts.

Reply:
@sandersn thx for taking a time to respond!
It's just a suggestion, so it's ok to merge this one without that change, thx!